server/src/main/java/org/elasticsearch/index/query/NestedQueryBuilder.java

```diff
@@ -76,7 +76,7 @@ public NestedQueryBuilder(String path, QueryBuilder query, ScoreMode scoreMode)
         this(path, query, scoreMode, null);
     }
 
-    private NestedQueryBuilder(String path, QueryBuilder query, ScoreMode scoreMode, InnerHitBuilder innerHitBuilder) {
+    public NestedQueryBuilder(String path, QueryBuilder query, ScoreMode scoreMode, InnerHitBuilder innerHitBuilder) {
```

Comment:
I needed to make this `public` so that I could programmatically create nested queries with inner hits